### PR TITLE
v0.5.0 doc changes and minor fixes

### PR DESCRIPTION
**Pull request by rodnymolina**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+  * Added config to strictly honor container capabilities from higher level container manager.
+  * Added support for per-container configs via `SYSBOX_*` env vars.
+  * Improved performance of Sysbox's syscall interception code.
+  * Improved the way Sysbox releases the seccomp-fd handles for intercept syscalls (kernels >= 5.8).
+  * Update to golang 1.16.
+  * Fixed segfault when building Docker image inside Sysbox container (issue 484).
+  * Fixed segfault when running python pip install inside nested sysbox container (issue 485).
+  * Fixed issue with running KinD inside a Sysbox container (issue 415).
+  * Fixed problem with shiftfs mounts on Kubernetes persistent volumes (issue 431).
+
```
Comment (felipecrs) [inline CHANGELOG.md:20]: @ctalledo, @rodnymolina I think you guys forgot to mention:

* Allow alternative docker data-dir inside a Sysbox container by reading `/etc/docke/daemon.json`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,26 @@
...
+  * Added configs to disable trapping chown and xattr* syscalls (improves performance but may reduce functionality).
+  * Added config to strictly honor container capabilities from higher level container manager.
+  * Added support for per-container configs via `SYSBOX_*` env vars.
+  * Improved performance of Sysbox's syscall interception code.
+  * Improved the way Sysbox releases the seccomp-fd handles for intercept syscalls (kernels >= 5.8).
+  * Update to golang 1.16.
+  * Fixed segfault when building Docker image inside Sysbox container (issue 484).
+  * Fixed segfault when running python pip install inside nested sysbox container (issue 485).
+  * Fixed issue with running KinD inside a Sysbox container (issue 415).
+  * Fixed problem with shiftfs mounts on Kubernetes persistent volumes (issue 431).
```
Comment (felipecrs) [inline CHANGELOG.md:19]:
```suggestion
  * Fixed problem with shiftfs mounts on Kubernetes persistent volumes (issue #431).
```

I believe GitHub will auto render a link to the issue this way.

File: CHANGELOG.md (hunk 1 above)
Comment (ctalledo) [reply CHANGELOG.md:21]: Done adding this.

**Comment by rodnymolina** [comment]
Closing this PR associated to the `v0.5.0-beta` branch. The new v0.5.0 release changes will be merged as part of the `release_v0.5.0` branch.

**Pull request closed without merging**